app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,15 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str, hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
```

Comment:
Consider using a more descriptive variable name for `self._hummer_level`. It seems like it might be a typo for `self._hammer_level`, which would be more descriptive and correct.